community/howto/voidtrance/Klipper_Macros_Beginners_Guilde.md

```diff
@@ -0,0 +1,467 @@
+# Writing Klipper Macros
```

Comment:
it's a silly thing, but this file should be "Guide" not "Guilde"

Reply:
Should be fixed with latest pull request.